components/console/changing_default_command.rst

```diff
@@ -9,20 +9,14 @@ name to the ``setDefaultCommand()`` method::
 
     use Symfony\Component\Console\Attribute\AsCommand;
     use Symfony\Component\Console\Command\Command;
-    use Symfony\Component\Console\Input\InputInterface;
-    use Symfony\Component\Console\Output\OutputInterface;
+    use Symfony\Component\Console\Style\SymfonyStyle;
 
-    #[AsCommand(name: 'hello:world')]
+    #[AsCommand(name: 'hello:world', description: 'Outputs "Hello World"')]
     class HelloWorldCommand extends Command
```

Comment:
Extends is required in this case ?

Reply:
It's optional, but for simplicity in this case (as this snippet is outside the framework scope) I kept it intentionally, otherwise we'd need to do: `$command = (new Command('hello:world'))->setCode(new HelloWorldCommand())->setDescription('...')`, then the attribute info is useless.